CRM/Contribute/Form/Task/PDFLetter.php

```diff
@@ -82,7 +84,7 @@ public function buildQuickForm() {
 
     // Contribute PDF tasks allow you to email as well, so we need to add email address to those forms
     $this->add('select', 'from_email_address', ts('From Email Address'), $this->_fromEmails, TRUE);
-    CRM_Core_Form_Task_PDFLetterCommon::buildQuickForm($this);
+    $this->addPDFElementsToForm();
```

Comment:
Line 83: CRM_Contact_Form_Task_PDFLetterCommon::buildQuickForm does exactly the same thing as line 85 CRM_Core_Form_Task_PDFLetterCommon::buildQuickForm(), so that's why it gives the deprecation notice.